modules/030-cloud-provider-azure/hooks/storage_classes.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2021 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hooks
+
+import (
+	"regexp"
+
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+	"github.com/flant/addon-operator/sdk"
+	"github.com/tidwall/gjson"
+)
+
+type StorageClass struct {
+	Name              string `json:"name"`
+	Type              string `json:"type"`
+	DiskIOPSReadWrite int64  `json:"diskIOPSReadWrite,omitempty"`
+	DiskMBpsReadWrite int64  `json:"diskMBpsReadWrite,omitempty"`
+}
+
+var storageClassesConfig = []StorageClass{
+	{
+		Name: "managed-standard-ssd",
+		Type: "StandardSSD_LRS",
+	},
+	{
+		Name: "managed-standard",
+		Type: "Standard_LRS",
+	},
+	{
+		Name: "managed-premium",
+		Type: "Premium_LRS",
+	},
+}
+
+var _ = sdk.RegisterFunc(&go_hook.HookConfig{
+	OnBeforeHelm: &go_hook.OrderedConfig{Order: 20},
+}, storageClasses)
+
+func storageClasses(input *go_hook.HookInput) error {
```

Comment:
Can we use library [hook](https://github.com/deckhouse/deckhouse/blob/main/go_lib/hooks/storage_class/hook.go) here?

Reply:
We cannot use it without modifications. This library is currently used only in yandex provider.

Reply:
ok.